debug_module.tex

```diff
@@ -52,18 +52,20 @@ \section{Debug Module Interface (DMI)} \label{dmi}
 \section{Reset Control} \label{reset}
 
 The Debug Module controls a global reset signal, which can
-reset, or hold in reset, every component in the platform,
+reset, or hold in reset, every component in the platform
 except for the Debug Module and Debug
 Transport Modules. The purpose of this feature is to allow debugging
 programs from the first instruction executed, so exactly what is affected
 by this reset is implementation dependent. The Debug Module's own state and registers are
-generally reset at power-up and if and only if
+generally reset at power-up if and only if
 \Fdmactive in \Rdmcontrol is 0. This means that the halt state of all harts is
 maintained provided that \Fdmactive is 1, although trigger CSRs may be cleared.
 
 Due to clock and power domain crossing issues,
 it may not be possible to perform arbitrary DMI accesses across reset,
-even if the Debug Module state is maintained. It must always be possible to access
+even if the Debug Module state is maintained. The debugger must not perform
+a reset while it's expecting a DMI access to complete. It must always be
```

Comment:
I don't see any reason to access the DM while reset is asserted (except for custom extensions, and people implementing those could build their system to not have this limitation). So how about:
While \Fndmreset or any external reset is asserted, the only supported DM operation is accessing \Rdmcontrol. The behavior of other accesses are undefined.